src/sysgpu/main.zig

```diff
@@ -23,6 +17,24 @@ const impl = switch (backend_type) {
 var inited = false;
 var allocator: std.mem.Allocator = undefined;
 
+fn backendFromBuildOption(comptime opt: build_options.@"build.SysgpuBackend") sysgpu.BackendType {
```

Comment:
This being named `build_options.@"build.SysgpuBackend"` looks weird and wrong to me.

We have other options in `build_options` that don't have to be written that way, for example here you can see we have `build_options.want_sysaudio` as a boolean: https://github.com/hexops/mach/blob/main/src/main.zig#L18

I think we do something similar for `core.PlatformType`: https://github.com/hexops/mach/blob/main/build.zig#L48

Can you look into making it work similar to that?

Reply:
I think the solution might be writing something like:

```
const backend_type: sysgpu.BackendType = switch(build_options.sysgpu_backend) {
    // ... all the cases ...
};
```

And then don't have a `backendFromBuildOption` function at all.

Reply:
Ah yeah that would be nicer. I've updated the PR :)